Cargo.toml

```diff
@@ -139,9 +139,13 @@ urlencoding        = "2.1.3"
 vfs                = "0.12.0"
 xxhash-rust        = "0.8.10"
 # oxc crates share the same version
-arcstr             = "1.2.0"
-lightningcss       = { version = "1.0.0-alpha.57" }
-oxc                = { version = "0.20.0", features = ["sourcemap_concurrent", "transformer", "minifier"] }
+arcstr = "1.2.0"
+lightningcss = { version = "1.0.0-alpha.57" }
+oxc = { git = "https://github.com/oxc-project/oxc", branch = "release/crates-1721271704", features = [
```

Comment:
Can you change to the version, using the git URL would invalidate the cache.

Reply:
Once the test passes or it is determined that the test failure has nothing to do with `oxc`, I will merge the release RP and replace it.